EKF/ekf_helper.cpp

```diff
@@ -1593,9 +1593,8 @@ void Ekf::setControlEVHeight()
 // and calculate a rotation matrix which rotates EV measurements into the EKF's navigation frame
 void Ekf::calcExtVisRotMat()
 {
-	// calculate the quaternion delta between the EKF and EV reference frames at the EKF fusion time horizon
-	Quatf quat_inv = _ev_sample_delayed.quat.inversed();
-	Quatf q_error =   quat_inv * _state.quat_nominal;
+	// calculate the quaternion delta between the EV and EKF reference frames at the EKF fusion time horizon
```

Comment:
// Calculate the quaternion delta that rotates from the EV to the EKF reference frame at the EKF fusion time horizon.